app/recipes/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe model"""
+    name = models.CharField(max_length=255)
```

Comment:
Since we use Postgres there is no downside to using `models.TextField` for all character fields.

Reply:
OK. Updated